filters.py

```diff
@@ -83,12 +83,12 @@ def _get_attribute_choices(self, attribute):
 
 
 class ProductCategoryFilter(ProductFilter):
+    categories = self.category.get_descendants(include_self=True)
```

Comment:
@Pacu2  now is better?

Reply:
I think you should take a different approach in resolving bugs.

I think that you could start the Saleor on your local instance and reproduce the situation listed in the issue to confirm that the bug appears in your local copy of Saleor.

Having a bug reproduced, you could make attempts to fix it, as in the more complex situation it might not be possible to fix it via the guessing game.

It's not better, as that would throw an error as `self` will be undefined at this point. You should define that `categories` in each function calling for an attributes lookup.